src/main/java/com/easypost/easyvcr/Censors.java

```diff
@@ -308,7 +306,7 @@ public static String applyUrlCensors(String url, String censorText,
         URI uri = URI.create(url);
 
         String path = Utilities.extractPathFromUri(uri);
-        Map<String, String> queryParameters = Utilities.queryParametersToMap(uri);
+        Map<String, String> queryParameters = Tools.queryParametersToMap(uri);
```

Comment:
Still MUCH prefer `utilities` to `tools` for code naming. A tool is something I'd run on the CLI. A utility is a code helper inside of a project.

At EasyPost, we use the `utilities` convention everywhere. We should try to stick to it.

Reply:
Utilities is publicly-accessible functions that end users can take advantage of.
Tools is internal-only functions that are not meant for public use.

Thoughts?

Reply:
I say we follow the pattern already established. In our libs we have Utils and InternalUtils. Why not follow that here?